src/htcondorce/registry.py

```diff
@@ -139,83 +139,85 @@ def create_app(test_config = None):
     else:
         app.config.update(test_config)
 
-    global _loader
+    global _jinja_env
     template_location = app.config.get('HTCONDORCE_TEMPLATES', '/usr/share/condor-ce/templates')
-    _loader = genshi.template.TemplateLoader(template_location, auto_reload=True)
+    _jinja_env = Environment(loader=FileSystemLoader(template_location),
+                             autoescape=select_autoescape(['html', 'xml']),
+                             auto_reload=True)
```

Comment:
```suggestion
                             autoescape=select_autoescape(['html', 'xml']))
```

I don't think we need to enable auto reload since we're loading static files on disk

Reply:
I set this to auto reload because that's the way it was working under Genshi. If you think we can disable it, that's fine with me.

Reply:
Ah good point. I don't think it's strictly necessary but I'm not interested in experimenting to find out here!